src/main/scala/com/linagora/openpaas/gatling/unifiedinbox/UnifiedInboxTemplateRequestsList.scala

```diff
@@ -299,19 +298,19 @@ object LoginTemplateRequestsList {
     "/linagora.esn.unifiedinbox.social/app/components/sidebar/social-networks-settings/social-networks-settings.html",
     "/views/modules/settings-overlay/template.html",
     "/account/api/accounts/providers",
-    "/unifiedinbox.twitter/app/components/sidebar/sidebar-menu.html",
-    "/unifiedinbox.github/app/components/sidebar/sidebar-menu.html",
+    //"/unifiedinbox.twitter/app/components/sidebar/sidebar-menu.html",
+    //"/unifiedinbox.github/app/components/sidebar/sidebar-menu.html",
     "/views/modules/search/provider-select/search-provider-select.html",
     "/views/modules/search/form/advanced/search-advanced-toggle-button.html",
     "/views/modules/user-notification/list/user-notification-list.html",
     "/views/modules/attachment/list/attachment-list.html",
     "/unifiedinbox/app/components/sidebar/folder-settings/folder-settings.html",
     "/unifiedinbox/views/partials/empty-messages/containers/inbox.html",
-    "/unifiedinbox.twitter/app/components/sidebar/sidebar-menu-item.html",
-    "/unifiedinbox.github/app/components/sidebar/sidebar-menu-item.html",
+    //"/unifiedinbox.twitter/app/components/sidebar/sidebar-menu-item.html",
+    //"/unifiedinbox.github/app/components/sidebar/sidebar-menu-item.html",
     "/views/modules/infinite-list/infinite-list.html",
     "/views/modules/attachment/list/item/attachment-list-item.html",
-    "/api/themes/53abdd2d5c5a7fc70a21f366/logo",
+    //"/api/themes/53abdd2d5c5a7fc70a21f366/logo",
```

Comment:
can we get rid of the comments? we have git for that :)

or is it a reminder that we should replace this by an other asset?